packages/main/src/MultiInput.ts

```diff
@@ -114,10 +51,71 @@ const metadata = {
  * @since 1.0.0-rc.9
  * @public
  */
+@customElement("ui5-multi-input")
+/**
+ * Fired when the value help icon is pressed
+ * and F4 or ALT/OPTION + ARROW_UP/ARROW_DOWN keyboard keys are used.
+ *
+ * @event sap.ui.webc.main.MultiInput#value-help-trigger
+ * @public
+ */
+@event("value-help-trigger")
+
+/**
+ * Fired when a token is about to be deleted.
+ *
+ * @event sap.ui.webc.main.MultiInput#token-delete
+ * @param {HTMLElement} token deleted token.
+ * @public
+ */
+@event("token-delete", {
+	detail: {
+		token: { type: HTMLElement },
+	},
+})
+
 class MultiInput extends Input {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Determines whether a value help icon will be visualized in the end of the input.
+	 * Pressing the icon will fire <code>value-help-trigger</code> event.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.Input.prototype.showValueHelpIcon
```

Comment:
`sap.ui.webc.main.Input.` should be replaced with `sap.ui.webc.main.MultiInput`.

Reply:
Done